modules/ray_tracing/doc/sqa_reports.yml

```diff
@@ -0,0 +1,39 @@
+Applications:
+    ray_tracing:
+        exe_directory: ${MOOSE_DIR}/modules/ray_tracing
+        app_types:
+            - RayTracingApp
+        content_directory: ${MOOSE_DIR}/modules/ray_tracing/doc/content
+        log_missing_markdown: WARNING
+
+Documents:
+    software_requirements_specification: sqa/ray_tracing_srs.md
+    software_design_description: sqa/ray_tracing_sdd.md
+    software_test_plan: sqa/ray_tracing_stp.md
+    requirements_traceablity_matrix: sqa/ray_tracing_rtm.md
+    verification_validation_plan: sqa/ray_tracing_vvr.md
+    log_safety_software_determination: WARNING
+    log_quality_level_determination: WARNING
+    log_enterprise_architecture_entry: WARNING
+    log_project_management_plan: WARNING
+    log_software_quality_plan: WARNING
+    log_configuration_management_plan: WARNING
+    log_asset_management_plan: WARNING
+    log_verification_validation_plan: WARNING
+    log_user_manual: WARNING
+    log_theory_manual: WARNING
+    log_software_requirements_specification: WARNING
+    log_software_design_description: WARNING
+    log_software_test_plan: WARNING
+    log_requirements_traceablity_matrix: WARNING
+
+Requirements:
+    ray_tracing:
+        directories:
+            - ${MOOSE_DIR}/modules/ray_tracing
+        log_missing: WARNING
+        log_missing_design: WARNING
+        log_missing_issues: WARNING
+        log_top_level_detail: WARNING
+        log_missing_detail: WARNING
+        log_testable: WARNING
```

Comment:
Same with these, if we don't need these then SQA is nearly done.

Reply:
```
ray_tracing WARNING
    log_duplicate_requirement: 0
                  log_missing: 0
           log_missing_design: 0
           log_missing_issues: 0
         log_top_level_detail: 0
           log_missing_detail: 0
                 log_testable: 1
   log_deprecated_requirement: 0
        log_deprecated_design: 0
        log_deprecated_issues: 0
        log_deprecated_detail: 0
  log_deprecated_verification: 0
    log_deprecated_validation: 0
  log_deprecated_with_details: 0
      log_missing_requirement: 0
        log_empty_requirement: 0
             log_empty_design: 0
             log_empty_issues: 0
       log_empty_verification: 0
         log_empty_validation: 0
             log_empty_detail: 0
        log_extra_requirement: 0
             log_extra_design: 0
             log_extra_issues: 0
             log_issue_format: 0
             log_design_files: 0
         log_validation_files: 0
       log_verification_files: 0
         log_duplicate_detail: 0 
```

Only one skip that needs to be fixed with a libMesh update

Reply:
Everything has been removed but `log_testable`. Keeping this here as a reminder to do it later when the test is no longer skipped